packages/featureflags/featureToggleManager.js

```diff
@@ -72,6 +75,22 @@ class FeatureToggleManager {
 
         return growthbook.isOn(featureName)
     }
+
+    async getFeatureValue (keystoneContext, featureName, defaultValue, featuresContext) {
+        const context = this.#getContext(keystoneContext)
```

Comment:
As far as I known this should be also done with await
```suggestion
        const context = await this.#getContext(keystoneContext)
```